UC-Berkeley-Extension/global
# Taxonomy template suggestion

## Asana Ticket(s)

## Description
- Adds page template suggestions based on taxonomy term id.
- Adds placeholder page template, header, and footer.

## Testing instructions
- `npm start` on theme base dir to compile scss
- `ddev start` on project root
- `ddev ssh`, `.. /scripts/refresh.sh` -> This should download content and taxonomy onto your local.
- Check out your site. You should have an article without taxonomy selected. (https://global.ddev.site/node/2) Ensure that the header color is Berkely blue.
<img width="500" alt="Screen Shot 2020-10-02 at 4 05 24 PM" src="https://user-images.githubusercontent.com/10488517/94970112-8e767e80-04c9-11eb-88fa-3fda44305167.png">

- Create an Article content. Mark this school as Haas on the select list. Ensure that the header color is now green.
<img width="500" alt="Screen Shot 2020-10-02 at 4 05 34 PM" src="https://user-images.githubusercontent.com/10488517/94970152-a6e69900-04c9-11eb-951c-e2c1f51fb53a.png">

- Also check the dev tools, and look at the template suggestions. Confirm that the suggested template names for the page are acceptable. It is attaching term id number at the end - in this case, Haas taxonomy term id is 1.
<img width="500" alt="Screen Shot 2020-10-02 at 4 10 21 PM" src="https://user-images.githubusercontent.com/10488517/94970240-d5fd0a80-04c9-11eb-999b-e89864e0597e.png">

- Confirm that adding the class name to the page wrapper (in this case, `haas`) is sufficient for "microsite" styling. Naming convention probably needs to be adjusted.
